src/commands/pack/macos.ts

```diff
@@ -31,11 +44,16 @@ export default class PackMacos extends Command {
     await writeScript('preinstall')
     await writeScript('postinstall')
     const args = [
-      '--root', buildConfig.workspace({platform: 'darwin', arch: 'x64'}),
-      '--identifier', c.macos.identifier,
-      '--version', buildConfig.version,
-      '--install-location', `/usr/local/lib/${config.dirname}`,
-      '--scripts', scriptsDir,
+      '--root',
```

Comment:
Maybe we could ignore the rule here since it's actually clearer to see these side-by-side